File: Checker source code with regex generated/Factorial check/test/tc2/tes2.pas
var a,b,d:integer;

(*
 * fungsi faktorial
 * param int n
 * return n!
 *)
function facs(n:integer):integer;
begin
	if (n=0) then
	begin
		facs := 1;
	end else
	begin
		facs := n*facs(n-1);
	end;
end;

begin
	// baca nilai a dan b
	readln(a,b); //comment wkwk
	// tulis nilai a + b
	writeln(a+b);
	{
		comment lagi
	}
	d := facs(a);
	writeln(a,' ',d);
end.

File: Checker source code with regex generated/Factorial check/test/tc2/tes2.cpp
#include <cstdio>
#include <cstdlib>
#include <iostream>

/*
 * fungsi faktorial
 * param int n
 * return n!
 */
int facs(int n){
	if (n==0) return 1;
	return n*facs(n-1);
}

int main(){
	int a,b;
	// baca nilai a dan b
	scanf("%d%d",&a,&b);//comment wkwk
	// tulis nilai a + b
	printf("%d\n",a+b);
	int d = facs(a);
	printf("%d %d\n",a,d);
	return 0;
}
